SFR-1410_ManifestScript.py

```diff
@@ -0,0 +1,49 @@
+import boto3
+import json
+import logging
+
+from urllib.parse import urlparse, parse_qsl, urlencode, urlunparse
+
+logging.basicConfig(filename='manifest.log', encoding='utf-8', level=logging.INFO)
+
+s3_client = boto3.client("s3")
+
+listObjects = s3_client.list_objects_v2(Bucket='drb-files-qa', Prefix='manifests/muse/')
+contentListObjects = listObjects['Contents']
+
+#Iterating through all the Content objects to access the JSON muse objects
+for c in contentListObjects:
+    currKey = c['Key']
+    museObject = s3_client.get_object(Bucket='drb-files-qa', Key=f'{currKey}')
```

Comment:
If this is something we are committing to the repo we should also have the `Bucket` name be configurable as a variable or something so that we can run this script on the production/qa buckets without having to change anything in the code